asreview/webapp/help.json

```diff
@@ -5,24 +5,24 @@
       "title": "Frequently asked questions",
       "items": [
         {
-          "title": "Has the use of ASReview in systematic reviews been validated?",
-          "url": "https://github.com/asreview/asreview/discussions/556"
+          "title": "What is the idea behind active learning?",
```

Comment:
This question sounds too general. How about "How does active learning help with systematic reviews?"

Reply:
Thanks. Good suggestion.